recommendations/richa-chicago.md

```diff
@@ -0,0 +1,5 @@
+## Things to do in Discover
+
+### This will be fun
```

Comment:
:tada: @richavekaria so excited to be here! Any recommendations for great Chinese restaurants?